src/Symfony/Component/Semaphore/composer.json

```diff
@@ -24,7 +24,8 @@
         "psr/log": "^1|^2|^3"
     },
     "require-dev": {
-        "predis/predis": "^1.1|^2.0"
+        "predis/predis": "^1.1|^2.0",
+        "symfony/lock": "^5.4 || ^6.0 || ^7.0"
```

Comment:
do we really want to allow those old versions here?

Reply:
We can change to everything the core team wants. Currently it works on this version and I used 5.4 as its still supported LTS until 2029.